graphql_api/types/bundle_analysis/base.py

```diff
@@ -135,6 +145,65 @@ def resolve_assets(
     return list(bundle_report.assets())
 
 
+@bundle_report_bindable.field("assetsPaginated")
+def resolve_assets_paginated(
+    bundle_report: BundleReport,
+    info: GraphQLResolveInfo,
+    ordering: AssetOrdering = AssetOrdering.SIZE,
+    ordering_direction: OrderingDirection = OrderingDirection.DESC,
+    first: Optional[int] = None,
+    after: Optional[str] = None,
+    last: Optional[int] = None,
+    before: Optional[str] = None,
+) -> Dict[str, object]:
+    # All filtered assets before pagination
+    assets = list(
+        bundle_report.assets(
+            ordering=ordering.value,
+            ordering_desc=ordering_direction.value == OrderingDirection.DESC.value,
+        )
+    )
+
+    total_count, has_next_page, has_previous_page = len(assets), False, False
+    start_cursor, end_cursor = None, None
+
+    # Apply cursors to edges
+    if after is not None:
```

Comment:
in the event after and before are both not None, this code will default to before; but I'm wondering if we should throw an error maybe?

Reply:
Just implementing off of the official spec for this style of pagination [here](https://relay.dev/graphql/connections.htm) it says:
>Including a value for both first and last is strongly discouraged, as it is likely to lead to confusing queries and results. The PageInfo section goes into more detail here.
With the current setup it does indeed become messed up when both are used. I agree with changing to throw an error if both are used at the same time.

Reply:
Done.